### DISCUSSION TITLE
Make form element hidden but still access its value at client side

### DISCUSSION THREAD

**Discussion by rhemasystems** [serenity-is/Serenity#6936] [Help]
Please how do I make the below highlighted controls and their caption invisible on a form but still be able to access its value in dialog.ts

![image](https://github.com/serenity-is/Serenity/assets/50826372/a6c504b9-b83b-4452-bc39-91b5abbfda99)

**Comment by luongnv6** [comment]
hide on dialog.ts:
```ts
this.form.Active.getGridField().hide();
```
hide on form.cs
```C#
        [Hidden]
        public boolean Active{ get; set; }
```
or just hide when Insert
```C#
        [HideOnInsert]
        public boolean Active{ get; set; }
```
or just hide when Update
```C#
        [HideOnUpdate]
        public boolean Active{ get; set; }
```


to get value:
```ts
var active = this.entity.Active;
//or
var acrive2 = this.form.Active.value;
```
